blivet/blivet.py

```diff
@@ -46,7 +46,7 @@
 from .flags import flags
 from .platform import platform as _platform
 from .formats import get_format
-from .osinstall import FSSet, find_existing_installations
+from .osinstall import FSSet, StorageDiscoveryConfig find_existing_installations
```

Comment:
It is missing (good catch, too), but I remove that entire line in 4ce6b9b, so it's moot.